liblangutil/DebugInfoSelection.cpp

```diff
@@ -39,6 +39,7 @@ DebugInfoSelection const DebugInfoSelection::All(bool _value) noexcept
 	DebugInfoSelection result;
 	for (bool DebugInfoSelection::* member: componentMap() | ranges::views::values)
 		result.*member = _value;
+	result.ethdebug = false;
```

Comment:
I don't want to interfere with the default debug selection mechanism. So I just excluded `ethdebug`from `All()`.

Reply:
You need to update the boolean `all()` predicate as well, because the way it is now, `DebugInfo::All().all()` returns false, which is very unintuitive and may actually result in a broken check somewhere.

But I generally feel that changing the meaning of seemingly obvious things like this is a footgun. It would be less error-prone not to hack it into `All()` but instead explicitly exclude `ethdebug` where appropriate. Just add something like `Except()` where you list only the members you want to exclude. Then explicitly use `Except(DebugInfoSelection::*ethdebug)` in places where we don't want experimental members. I might actually have a use for `Except()` in my PR and was considering introducing it myself.